### PR TITLE
added Zed2i ros bag recorder. it is tested on anton ego

### PR DESCRIPTION
**Pull request by cameron-a-johnson**

### CODE REVIEW

File: tmux/record/record_ros_bag_Zed2i.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  #           -p head_pose_topic:=HeadsetPoseData
+  #           -p pv_width:=1280
+  #           -p pv_height:=720
+  #           -p pv_framerate:=30
+  #           -p sm_freq:=5
+  #           -p rm_depth_AHAT:=disable
+  #       # Visualize Images being output from the headset
+  #       - rqt_pv_images: rqt -s rqt_image_view/ImageView
+  #           --args ${ROS_NAMESPACE}/PVFramesBGR
+  #           --ros-args -p _image_transport:=raw
```
Comment (Purg) [inline tmux/record/record_ros_bag_Zed2i.yml:92]: This could be removed.

File: tmux/record/record_ros_bag_Zed2i.yml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            ros_params_override_path:=${ANGEL_WORKSPACE_DIR}/config/zed_params_override.yaml
+        # Enable sending ROS2 messages to receiving agents in an HoloLens2 app.
+        - datahub: ros2 run ros_tcp_endpoint default_server_endpoint  --ros-args
+            -r __ns:=${ROS_NAMESPACE}
+            -p ROS_IP:=0.0.0.0
+        # Trim out the timestamp of the image topic of choice.
+        - run_image_timestamp: ros2 run angel_system_nodes image_timestamp_relay --ros-args
+            -r __ns:=${ROS_NAMESPACE}
+            -p image_topic:="${ZED_IMAGE_TOPIC}"
+            -p output_topic:="${ZED_IMAGE_TS_TOPIC}"
```
Comment (Purg) [inline tmux/record/record_ros_bag_Zed2i.yml:112]: I would say that we don't need to create and record the timestamp messages. This node and output is trivial to run at system runtime.

File: tmux/record/record_ros_bag_Zed2i.yml (hunk 1 above)
Comment (cameron-a-johnson) [reply tmux/record/record_ros_bag_Zed2i.yml]: Removed, thanks.

File: tmux/record/record_ros_bag_Zed2i.yml (hunk 2 above)
Comment (cameron-a-johnson) [reply tmux/record/record_ros_bag_Zed2i.yml]: Removed the timestamp recording, thanks.

Comment (Purg) [reply tmux/record/record_ros_bag_Zed2i.yml]: I meant that the whole `run_image_timestamp` command could be removed, and then the `"${ROS_NAMESPACE}/${ZED_IMAGE_TS_TOPIC}"` topic being recorded may be removed.

File: tmux/record/record_ros_bag_Zed2i.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  # Changing the domain ID was important at KHQ to unblock perceived network
+  # congestion slowdowns to message sending.
+  export ROS_DOMAIN_ID=77
+
+  # Topic to use from those emitted from the ZED Wrapper launch stack.
+  # This name is *NOT* prepended with the common namespace.
+  export ZED_IMAGE_TOPIC="zed/zed_node/rgb_raw/image_raw_color"
+  # Expanding on the above, this is the topic for the timestamp extraction of
+  # the images used.
+  export ZED_IMAGE_TS_TOPIC="${ZED_IMAGE_TOPIC}_ts"
```
Comment (Purg) [inline tmux/record/record_ros_bag_Zed2i.yml:33]:
```suggestion
```

File: tmux/record/record_ros_bag_Zed2i.yml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+            camera_model:=zed2i
+            ros_params_override_path:=${ANGEL_WORKSPACE_DIR}/config/zed_params_override.yaml
+        # Enable sending ROS2 messages to receiving agents in an HoloLens2 app.
+        - datahub: ros2 run ros_tcp_endpoint default_server_endpoint  --ros-args
+            -r __ns:=${ROS_NAMESPACE}
+            -p ROS_IP:=0.0.0.0
+        # Trim out the timestamp of the image topic of choice.
+        - run_image_timestamp: ros2 run angel_system_nodes image_timestamp_relay --ros-args
+            -r __ns:=${ROS_NAMESPACE}
+            -p image_topic:="${ZED_IMAGE_TOPIC}"
```
Comment (Purg) [inline tmux/record/record_ros_bag_Zed2i.yml:85]:
```suggestion
```

File: tmux/record/record_ros_bag_Zed2i.yml [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+  - record:
+      layout: even-vertical
+      root: ./ros_bags
+      panes:
+        - ros_bag_record: ros2 bag record
+            "${ROS_NAMESPACE}/${ZED_IMAGE_TOPIC}"
+            "${ROS_NAMESPACE}/${ZED_IMAGE_TS_TOPIC}"
```
Comment (Purg) [inline tmux/record/record_ros_bag_Zed2i.yml:93]:
```suggestion
```

File: tmux/record/record_ros_bag_Zed2i.yml (hunk 2 above)
Comment (Purg) [reply tmux/record/record_ros_bag_Zed2i.yml]: Translated intent to suggestions below.

**Review by Purg — APPROVED** [review]

**Pull request merged**